alternator/conditions.cc

```diff
@@ -163,6 +164,14 @@ static bool check_IN(const rjson::value* val, const rjson::value& array) {
     return have_match;
 }
 
+// Check if a JSON-encoded value is null.
+static bool check_NULL(const rjson::value* val, const rjson::value& array) {
+    if (array.Size() > 0) {
```

Comment:
While this is of course correct, I wanted to comment that it is becoming confusing that some check_* functions are given an array and check its length - while others assume the length was already checked for them. I think it will be clearer if check_NULL's caller checked for an empty array, and check_NULL() would not get a second argument at all.